src/applications/ivc-champva/10-7959C/containers/ConfirmationPage.jsx

```diff
@@ -129,9 +138,12 @@ export function ConfirmationPage(props) {
           text="Print this page"
         />
       </div>
-      <a className="vads-c-action-link--green" href="https://www.va.gov/">
-        Go back to VA.gov
-      </a>
+      <h2 className="vads-u-font-size--h3">What to expect next</h2>
+      <p>
+        We’ll contact the number you listed on this form by mail or phone if we
+        have questions or need more information.
+      </p>
+      <VaLinkAction href="/" text="Go back to VA.gov" />
```

Comment:
Updates position of some copy and replaces link with newer `VaLinkAction` component.